SpringBootStarterProject/src/main/java/com/nous/project/template/controller/ProductController.java

```diff
@@ -19,6 +22,13 @@
 
 	@RequestMapping(method = RequestMethod.GET, produces = "application/json")
 	public List<Product> getAll() {
+		logerMethod();
 		return null;
 	}
+	
+	 private static final Logger log = (Logger) LoggerFactory.getLogger(ShoppingCartController.class);
+
+	    public void logerMethod() {
+	       log.info("Hello world");

```

Comment:
there should logger to handle error. Create a try catch block for exception and write log.error()